spk/radarr/src/service-setup.sh

```diff
@@ -9,20 +9,15 @@ SPK_RADARR="${SYNOPKG_PKGINST_TEMP_DIR}/share/Radarr/Radarr.exe"
 CONFIG_DIR="${SYNOPKG_PKGDEST}/.config"
 PID_FILE="${CONFIG_DIR}/Radarr/nzbdrone.pid"
 
-GROUP="sc-media"
+# Radarr uses the home directory to store it's ".config"
+# But there is a problem:
+# On DSM5 this is: PKG/var/.config/
+# On DSM6 this is: PKG/.config/
+# So we have to do upgrade magic
```

Comment:
Some request as for Sonarr. Please keep it simple and change `HOME` at process startup.

Reply:
Ah, I didn't know that was possible. I hope Sonarr listens to that variable. Will test.
Wonder why that approach wasn't taken before when they did this overhaul: https://github.com/SynoCommunity/spksrc/commit/5a0b6dabf2c86b4ea3ec50fb80aba1f93c9f7456